utilities.py

```diff
@@ -8,10 +9,10 @@ def parse_sentence(user_input):                               #returns root word
     import os
     from nltk.parse.stanford import StanfordDependencyParser
     import config
-    path_to_jar = config.stanford_path_to_jar
-    path_to_models_jar = config.stanford_path_to_models_jar
+    path_to_jar = Path(config.stanford_path_to_jar)
+    path_to_models_jar = Path(config.stanford_path_to_models_jar)
     dependency_parser = StanfordDependencyParser(path_to_jar=path_to_jar, path_to_models_jar=path_to_models_jar)
-    os.environ['JAVAHOME'] = config.javahome
+    os.environ['JAVAHOME'] = Path(config.javahome)
```

Comment:
@hritikpapa Did you try running this after making these changes?

Reply:
@hritikpapa this is not working

Reply:
@vishakha-lall  ca you please elaborate the issue you are facing, because I tried executing it and it's not throwing any errors